lib/main.js

```diff
@@ -54,6 +58,14 @@ const _definitions = [
         selector: ["www.pixiv.net"],
         fix: ["std-customElements"]
     },
+    {
+        selector: ["www.redditstatic.com/desktop2x/CommentsPage.*.js$script"],
+        fix: ["$script", "reddit-comments-regexp"]
+    },
+    {
+        selector: ["www.youtube.com"],
+        fix: ["std-customElements"]
```

Comment:
Youtube still deploys their own customElements Polyfill (all of webcomponents, actually), why is this required?
Adding this is much slower.